src/main/java/cms/gov/madie/measure/services/MeasureService.java

```diff
@@ -315,19 +315,9 @@ public Page<MeasureListDTO> getMeasures(
   }
 
   public void checkDuplicateCqlLibraryName(String cqlLibraryName) {
-    if (StringUtils.isNotEmpty(cqlLibraryName)
-        && measureRepository.findByCqlLibraryName(cqlLibraryName).isPresent()) {
-      throw new DuplicateKeyException(
-          "cqlLibraryName", "CQL library with given name already exists.");
-    }
-  }
-
-  public void checkDuplicateCqlLibraryName(String cqlLibraryName, String measureSetId) {
```

Comment:
Sorry Ethan, did not notice this before but looks like this unused method. can be removed.